src/applications/vaos/appointment-list/components/ExpressCareDetailsPage.jsx

```diff
@@ -87,7 +87,7 @@ function ExpressCareDetailsPage({
   }
 
   const unableToReachVeteran =
-    appointment.cancelationReason?.text === UNABLE_TO_REACH_VETERAN_DETCODE;
+    appointment.cancelationReason === UNABLE_TO_REACH_VETERAN_DETCODE;
```

Comment:
It seemed unnecessary to have this extra `text` object. The VAOS service has a cancellation reason without this.